src/components/ArticleCard/ArticleCard.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { PureComponent } from "react";
+import { WithStyles } from "@material-ui/core/styles/withStyles";
+import styles from "./styles";
+import { Card, CardContent, Typography, CardActions, Avatar, CardHeader, Chip, IconButton, withStyles } from "@material-ui/core";
+import Article from "../../interfaces/Article";
+import moment from "moment";
+import Favorite from "@material-ui/icons/Favorite";
+import Badge from '@material-ui/core/Badge';
+
+interface ArticleCardProps {
+    article: Article
+}
+
+const StyledBadge = withStyles(() => ({
+    badge: {
+      top: '50%',
+      right: -3,
+      fontSize: "0.8rem"
+    },
+}))(Badge);
+
+export class ArticleCard extends PureComponent<ArticleCardProps & WithStyles<typeof styles>> {
+	render() {
+		return (
+            <Card className={this.props.classes.card}>
+                <CardHeader
+                    avatar={
+                        <Avatar aria-label="Recipe" className={this.props.classes.avatar}>
+                            { this.props.article.author.username? this.props.article.author.username[0]: "" }
```

Comment:
try using es6 object destruction...
also you can use dafault props to avoid these kind of checks